shared/xccdf/system/software/updating.xml

```diff
@@ -249,6 +249,17 @@ NOTE: For U.S. Military systems, this requirement does not mandate DoD certifica
 this purpose; however, the certificate used to verify the software must be from an 
 approved Certificate Authority.
 </rationale>
+<warning category="general">
+Automated remediation for this configuration check is temporarily disabled.
+Enabling gpgcheck for repository metadata will disable installation of
+software from repositories that do not sign the metadata. Official Red Hat
+Enterprise Linux repositories do not provide a repository metadata signature.
+As a consequence, setting <tt>repo_gpgcheck</tt> to <tt>1</tt> on a typical
+system prevents <tt>yum</tt> from installation of software packages, including
+security updates and packages required by this guide.
+This issue of Red Hat Enterprise Linux repositories was reported in Red Hat
+Bugzilla #1410638.
```

Comment:
@jan-cerny use `<weblink-macro/>` for example
`<weblink-macro link="http://www.postfix.org/SMTPD_ACCESS_README.html#danger"/>`